src/System.Runtime.InteropServices.RuntimeInformation/src/project.json

```diff
@@ -6,10 +6,11 @@
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
-    "System.Threading": "4.0.0"
+    "System.Threading": "4.0.0",
+    "System.Reflection.Extensions": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "netstandard1.3": {
```

Comment:
Right, I misspoed about that.  Net45 and Win8 are actually 1.1, wpa8.1 is 1.2, it was wp8 that's 1.0.  I had an off-by-one error.  Carry on.